sunpy/image/transform.py

```diff
@@ -115,7 +116,8 @@ def _get_transform_method(method, use_scipy):
     return method
 
 
-def add_rotation_function(name, handles_clipping, handles_image_nans, handles_nan_missing):
+def add_rotation_function(name, allowed_orders,
```

Comment:
```suggestion
def add_rotation_function(name, *, allowed_orders,
```
We should definitely make the rest of these arguments keyword only

Reply:
Note that these parameters are not keyword parameters and are non-optional, so there's no such thing as making them keyword-only.  These parameters are all positional – as suggested by @Cadair! – so that every function is required to state `True`/`False` for each of these functionalities, with no default value.

All usages of the decorator still name the parameters as if they were keyword parameters, both for clarity and to seamlessly support if we ever do make them optional.

Reply:
Never mind.  I had an incomplete understanding of how the star worked.

Reply:
Fixed